arlo-client/src/components/Audit/index.test.tsx

```diff
@@ -207,9 +207,15 @@ describe('RiskLimitingAuditForm', () => {
 })
 
 describe('AA setup flow', () => {
+  // MultiJurisdictionAudit will only be rendered once the user is logged in, so
+  // we simulate that.
+  const MultiJurisdictionAuditWithAuth: React.FC = () => {
+    const { isAuthenticated } = useContext(AuthDataContext)
+    return isAuthenticated ? <MultiJurisdictionAudit /> : null
```

Comment:
is `null` the right alternate return value here, or should it be an empty react fragment?

Reply:
https://reactjs.org/docs/conditional-rendering.html#preventing-component-from-rendering